### ISSUE TITLE
Kubernetes Solution Update: Allow 3-Masters (and more) Deployment

### ISSUE THREAD

**Issue by mik-tf** [threefoldtech/tfgrid-sdk-ts#3757]
### Which package/s are you suggesting this feature for?

Dashboard

### Is your feature request related to a problem? Please describe

As discussed with Thabet and Scott, the current dashboard Kubernetes currently allows only for 1-master deployment, and updating it to allow for 3-masters and more would allow for HA on the grid for apps.

### Describe the solution you'd like

We propose to update the Kubernetes solution to allow for deployments with 3-masters (and more).

I.e., we want multiple master deployments for Kubernetes. 3 masters is generally considered a good starting point for HA.

This will enable high availability for apps on the grid.

### Proposition of Configurations

- Minimum: 3 masters, 3 workers
- Basic: 3 masters, 3-5 workers
- Advanced: 5 masters, 5-20 workers

The basic configuration would be a good start.

### Link with GlusterFS + CSI

I think we can add container storage interface ([CSI](https://github.com/container-storage-interface/spec)) with GlusterFS to allow for HA for app (kubernetes) and HA for storage (glusterFS). Might be for a separate issue once GlusterFS is done on Dashboard too (feature request: https://github.com/threefoldtech/tfgrid-sdk-ts/issues/3759)

**Comment by mik-tf** [comment]
As discussed with Thabet and Kristof, we will first start by having guides instead of dashboard solutions.
